packages/domains-table/src/domains-table/domains-table.tsx

```diff
@@ -363,6 +367,13 @@ export const DomainsTable = ( props: DomainsTableProps ) => {
 		Object.values< DomainData[] >( fetchedSiteDomains ).flat()
 	);
 
+	const currentUsersOwnsAllSelectedDomains = ! Array.from( selectedDomains ).some( ( selected ) =>
+		( domains || [] ).find(
```

Comment:
Let's use the null-coalescing operator instead of OR. It makes little difference in this context, but it's possible to understand immediately that the `domains` table might not exist (`??`) instead of it being falsy.

```suggestion
		( domains ?? [] ).find(
```